src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
```

Comment:
I was imagining the other coroutines would be similarly updated. Does that sound reasonable?